### PR TITLE
feat(dca): add provitional faq and landing

### PR DESCRIPTION
**Pull request by FiboApe**

### CODE REVIEW

File: apps/root/src/pages/dca/create-position/components/landing/chart.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        fill="#978AA8"
+      />
+      <line x1="52" y1="108.741" x2="577" y2="108.741" stroke="#A399FF" strokeWidth="3" strokeDasharray="15 15" />
```
Comment (th0rOdinson) [inline apps/root/src/pages/dca/create-position/components/landing/chart.tsx:25]: This should be violet600 (themeMode-based)

File: apps/root/src/pages/dca/create-position/components/landing/chart.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+      <defs>
+        <linearGradient
+          id="paint0_linear_6629_32270"
+          x1="595.399"
+          y1="88.6885"
+          x2="595.399"
+          y2="265.349"
+          gradientUnits="userSpaceOnUse"
+        >
+          <stop stopColor="#791AFF" />
```
Comment (th0rOdinson) [inline apps/root/src/pages/dca/create-position/components/landing/chart.tsx:123]: We can also replace this with violet500, it's the same in both modes, but it doesn't hurt to handle it this way 😄

File: apps/root/src/pages/dca/create-position/components/landing/index.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  const intl = useIntl();
+
+  const logoProps = { size: spacing(16), fill: colors[palette.mode].typography.typo2 };
+  const logo =
+    palette.mode === 'light' ? <BalmyLogoSmallDark {...logoProps} /> : <BalmyLogoSmallLight {...logoProps} />;
+
+  return (
+    <StyledBackgroundPaper variant="outlined">
+      {logo}
+      <Typography variant="h4" fontWeight={700}>
```
Comment (th0rOdinson) [inline apps/root/src/pages/dca/create-position/components/landing/index.tsx:64]: It looks better with textAlign center imho

File: apps/root/src/pages/dca/frame/index.tsx [hunk 4]
```diff
@@ -68,14 +67,14 @@ const DcaFrame = ({ isLoading }: DcaFrameProps) => {
           <ContainerBox flexDirection="column" gap={8}>
             <NetWorth walletSelector={{ options: { setSelectionAsActive: true } }} />
             {currentRoute === DCA_CREATE_ROUTE.key ? (
-              <CreatePosition handleChangeNetwork={handleChangeNetwork} />
+              <>
+                <CreatePosition handleChangeNetwork={handleChangeNetwork} />
+              </>
```
Comment (th0rOdinson) [inline apps/root/src/pages/dca/frame/index.tsx:72]: No need for fragment here

File: apps/root/src/pages/dca/create-position/components/landing/chart.tsx (hunk 1 above)
Comment (FiboApe) [reply apps/root/src/pages/dca/create-position/components/landing/chart.tsx:25]: You sure? I exported the svg directly from figma and it exported the same one both for dark and light

File: apps/root/src/pages/dca/create-position/components/landing/chart.tsx (hunk 2 above)
Comment (FiboApe) [reply apps/root/src/pages/dca/create-position/components/landing/chart.tsx:123]: Same as before

File: apps/root/src/pages/dca/create-position/components/landing/chart.tsx (hunk 1 above)
Comment (th0rOdinson) [reply apps/root/src/pages/dca/create-position/components/landing/chart.tsx:25]: Maybe i'm too meticulous at this point 😅 But this is the dashed line, and in light mode is looking much brighter comparing it with the design. I find this particular vector to change based on the theme

**Review by th0rOdinson — APPROVED** [review]

**Pull request merged**